### PR TITLE
Document list view

### PR DESCRIPTION
**Pull request by cbeer**
Fixes https://github.com/sul-dlss/spotlight/issues/910

Blocked by https://github.com/projectblacklight/blacklight/pull/1137

screen shot 2015-03-11 at 11 34 13 am

### CODE REVIEW

File: app/assets/stylesheets/blacklight.scss [hunk 1]
```diff
@@ -37,3 +37,30 @@
...
+
+  .index_title {
+    font-size: $font-size-large;
+  }
+
+  .document-counter {
+    display: none;
+  }
+
+  .document-thumbnail {
```
Comment (cbeer) [inline app/assets/stylesheets/blacklight.scss]: Is there a better way to position the image there (without changing the markup..)? Other things I tried were foiled by the `<dl>` `clear: left` styling.

Comment (cbeer) [inline app/assets/stylesheets/blacklight.scss]: Another approach that worked (but I'm not sure is better) was:

```css
  .index_title {
    margin-top: 0;
    font-size: $font-size-large;
    margin-left: 120px;
  }

  .document-counter {
    display: none;
  }

  .document-metadata {
    margin-left: 120px;
  }

  .document-thumbnail {
    float: left;
    padding: 0;
    margin: 0;
    height: 0;

    img {
      border: 1px solid $gray;
    }
  }
```

Comment (ggeisler) [inline app/assets/stylesheets/blacklight.scss]: In doing my mockup, I couldn't figure out how to do it without changing the markup. Though the end result of your approach looks good to me.

Comment (cbeer) [inline app/assets/stylesheets/blacklight.scss]: @jkeck any preference on CSS?

Comment (jkeck) [inline app/assets/stylesheets/blacklight.scss]: I think positioning may be the least obtrusive option w/o updating the markup.

Comment (cbeer) [inline app/assets/stylesheets/blacklight.scss]: Positioning with `absolute`, or with the 0-height float?

**Pull request merged**